packages/bundler/userOperation/v0_7.ts

```diff
@@ -259,6 +259,14 @@ export class UserOperationV0_7 {
     }
   }
 
+  public setPaymasterAndData(paymasterAndData: HexString) {
+    if (ethers.dataLength(paymasterAndData) < 20) {
+      return
+    }
```

Comment:
After testing, there's an issue:
when we switch the `Paymaster Option` from `VerifyingPaymaster` to `NullPaymaster` on the `Authorization page`, the `requestPaymasterAndData()` function returns a `0x` value, causing an immediate `return` without any effect.
It might be:
```
    if (ethers.dataLength(paymasterAndData) < 20) {
      this.paymaster = ""
      this.paymasterData = "0x"
      return
    }
```